rfq/negotiator.go

```diff
@@ -13,6 +14,18 @@ import (
 	"github.com/lightningnetwork/lnd/routing/route"
 )
 
+const (
+	// defaultRateTickExpirySeconds is the default rate tick expiry lifetime
+	// in seconds.
+	//
+	// TODO(ffranr): This const is currently used in conjunction with the
+	//  AcceptAllPrices flag. It is used to set the expiry time of the rate
+	//  tick in the accept message. This is a temporary solution and should
+	//  be replaced with an expiry time provided by the peer in the quote
```

Comment:
IIRC, we already have the expiry included in the message? 

I think for the final release, we want this to be more like 1 - 10 minutes.

Reply:
The RFQ quote request message type does not include an expiry currently. So the suggested rate tick is provided without an expiry.